snyk/cli
# feat: do not download iac rules bundle in cli anymore [CCSC-345]

#### What does this PR do?

This PR removes the code that was downloading and caching a rules bundle locally. This is now responsibility of `snyk-iac-test`, where the preferred bundle is downloaded and verified. 
We are leaving the env variable `IAC_BUNDLE_PATH` in place, so that it can be used for local development and debugging.


#### How should this be manually tested?

until it's merged, it can be verified with this PR: https://github.com/snyk/snyk-iac-test/pull/139

1. Build the snyk-iac-test binary locally.
2. Build the cli locally
3. Run `SNYK_IAC_POLICY_ENGINE_PATH=~/snyk-repos/snyk-iac-test/snyk-iac-test snyk-dev iac test test/fixtures/iac/terraform -d ` and check the logs - bundle should be downloaded in snyk-iac-test
4. To try a local bundle, create one locally first.
5. Then Run `SNYK_IAC_POLICY_ENGINE_PATH=.../snyk-repos/snyk-iac-test/snyk-iac-test SNYK_IAC_BUNDLE_PATH=.../snyk-repos/opa-rules/dist.tar.gz snyk-dev iac test test/fixtures/iac/terraform -d`  and check the logs noting the local bundle.

#### What are the relevant tickets?

CCSC-345
